app/views/profiles/_appointments.html.erb

```diff
@@ -0,0 +1,40 @@
+<% @patient = Patient.find_by medwallet_id: @user.id  %>
+<% @careplan = CarePlan.where patient_id: @patient.id %>
+<div class="table-responsive-xl">
+  <table class="table">
+    <thead>
+      <tr>
+        <th scope="col">Date</th>
+        <th scope="col">Status</th>
+        <th scope="col">Provider</th>
+        <th scope="col">Treatment</th>
+        <th scope="col">Cost</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @careplan.each do |careplan| %>
+        <tr>
+          <td id="careplan_<%=careplan.id%>_date">
+            <%= careplan.date %>
+          </td>
+          <td id="careplan_<%=careplan.id%>_status">
+            <% @status = Status.where id: careplan.status_id %>
+            <%= link_to careplan.status.name, "care_plans/" + careplan.status_id.to_s + "/edit", remote: true %>
```

Comment:
You can use route helpers to condense this a little

```suggestion
            <%= link_to careplan.status.name, edit_care_plan_path(careplan), remote: true %>
```

Additionally, when I clicked this link it _doesn't_ submit a JS request and does a regular full page load despite having the `remote: true` option.

The reason this happens is because you are missing

```erb
<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
from your `application.html.erb` file. This loads all the JS from the `/javascript/packs/` folder which allows the `link_to` and `form_with` methods to make JS requests instead of HTML requests. This might explain the difficulty you had using AJAX.

This method should've been present in the starting point— perhaps it was removed by mistake when Bootstrap was added?

Reply:
Good catch.